src/client/v2/indexer/lookupAccountTransactions.ts

```diff
@@ -307,7 +307,8 @@ export default class LookupAccountTransactions extends JSONRequest {
    * @category query
    */
   currencyGreaterThan(greater: number) {
-    this.query['currency-greater-than'] = greater;
+    if (greater === 0) this.query['currency-greater-than'] = '0';
+    else this.query['currency-greater-than'] = greater;
```

Comment:
What do you think about avoiding the condition and doing something like this?
```suggestion
    this.query['currency-greater-than'] = '' + greater;
```

or

```suggestion
    this.query['currency-greater-than'] = greater.toString();
```